docs/core/dependency-loading/understanding-assemblyloadcontext.md

```diff
@@ -84,9 +84,7 @@ When two <xref:System.Runtime.Loader.AssemblyLoadContext> instances contain type
 
 To complicate matters, exception messages about these mismatched types can be confusing. The types are referred to in the exception messages by their simple type names. The common exception message in this case would be of the form:
 
-```
-Object of type 'IsolatedType' cannot be converted to type 'IsolatedType'.
-```
+> Object of type 'IsolatedType' cannot be converted to type 'IsolatedType'.
```

Comment:
it's probably good to prevent this from being localized

```suggestion
> `Object of type 'IsolatedType' cannot be converted to type 'IsolatedType'.`
```

Reply:
Shouldn't we expect the exceptions provided by core to be localized? I think it would be save to localize it, but the variable names shouldn't be as those will always be the same.

```suggestion
> Object of type `IsolatedType` cannot be converted to type `IsolatedType`.
```

Reply:
I would vote for @Thraka's version but let me know what I apply as a suggested change, please

Reply:
If they are, then it should pick the TM from the software translation. @JasonCard can you confirm? Also, would we keep the sentence with no formatting so it can 100% match?

Reply:
@mairaw, It seems that blockquotes aren't localized by default. See the current french version:

![image](https://user-images.githubusercontent.com/31348972/66634551-9af52d80-ec0d-11e9-9201-e25c1f6c75ab.png)

so I think your suggestion and @Thraka one's doesn't make any big difference.

If it should be localized, then perhaps it shouldn't be blockquote. (Unless I'm missing something)

Reply:
Blockquotes don't really make a difference for localization. Let's leave this one as-is. Resolving this one.

Reply:
@mairaw sorry for the delayed reply. Actually @Youssef1313 is correct that blockquote (code block) is also blocked from localization. So if you want the exception message to be localized, you should remove the blockquote and use @Thraka 's suggestion. Although the exception message should be localized as part of SW product, translation may not be recycled unless the string is exactly the same as @maira mentioned.